js/education.js

```diff
@@ -0,0 +1,1914 @@
+// slick.min.js
+!(function (i) {
```

Comment:
you can remove slick js , since you have already added in the previous landing page